docs/getting-started/components/registry.md

```diff
@@ -1,31 +1,47 @@
 # Registry
 
-The Feast feature registry is a central catalog of all the feature definitions and their related metadata. It allows data scientists to search, discover, and collaborate on new features.
```

Comment:
I actually like this sentence as it helps give a high level understanding of it. Can we keep it?